sass/true/_assert.scss

```diff
@@ -153,6 +153,31 @@
   @include _true-assert-results($assert, $expected, 'unequal');
 }
 
+// Assert Error
+// ------------
+/// Assert that a mixin or function triggers an `@error`.
+///
+/// @group api-assert-values
+///
+/// @param {string} $message -
+///   Optional message to check against the emitted message.
+///   The emitted message must contain the expected message,
+///   allowing for partial message validation.
+///   This is a case-insensitive check.
+///
+///
+/// @example scss -
+///   @include test('Some mixin does what it is supposed to do') {
+///     @include assert-error('must pass valid params'){
+///       @include some-mixin($bad-params);
+///     }
+///   }
+@mixin assert-error($message: null) {
```

Comment:
I think we can improve messaging in the plain-CSS compile. Right now if you run `yarn compile` or `yarn commit`, this is the output:

```css
/* # Module: Assert Error */
/* ---------------------- */
/* Test: Detects when an error is thrown */
/*  */
/* Test: Compares a partial message to the emitted message (case-insensitive) */
/*  */
/* Test: Compares the full message to the emitted message (case-insensitive) */
/*  */
/*  */
```

I'd like to see something more like:

```css
/* Test: Compares the full message to the emitted message (case-insensitive) */
/*    ? [assert-error] Must pass a valid color */
/*  */
```

Reply:
Thanks for the feedback. I'll see what I can do. I was not evaluating the final css.